features/cli/default.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Default CLI command
+
+  In order to quickly use and understand AWS Session Credentials
+  As a AWS Session Credentials user
+  I want the default command to make sensible choices
+
+  Scenario: First time use
```

Comment:
Great! I was thinking of a `sudo` like behaviour.
It may also be worth noting that when:
* Getting a user session token - [max time is 36 hours](http://docs.aws.amazon.com/STS/latest/APIReference/API_GetSessionToken.html)
* Getting a role session token - [max time is 1 hour](http://docs.aws.amazon.com/STS/latest/APIReference/API_GetSessionToken.html)